config.py

```diff
@@ -66,6 +67,13 @@ def validate_config(config):
         ):
             raise ConfigError("Config error: Incorrect media reference settings")
 
+    if config.driver == "google_drive" and not (
```

Comment:
I wonder if we should not move these kinds of validations to driver class (e.g. validate() or even to init)

Reply:
I dont think that is a good idea, this allows media-sync to fail before doing anything. The required parameters are handled like for **minio**  as well and db-sync also handles some parameters like this.

Reply:
I meant more like check general config, then create driver instance and validate. But sure, we can leave it as is.